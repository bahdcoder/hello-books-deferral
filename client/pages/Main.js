import React from 'react';
import Home from '@pages/Home/index';
import Navbar from '@components/Navbar';
import { BrowserRouter, Route } from 'react-router-dom';
import ForgotPassword from '@home/ForgotPassword';
import ResetPassword from '@home/ResetPassword';

export default function Main() {
    return (
        <React.Fragment>
            <div className="md:pl-3 lg:pl-3 xl:pl-3 md:pr-3 lg:pr-3 xl:pr-3">
                <BrowserRouter>
                    <Navbar />
                    <Route exact path="/" component={Home} />
                    <Route path="/forgot" component={ForgotPassword} />
                    <Route path="/reset" component={ResetPassword} />
                </BrowserRouter>
            </div>
        </React.Fragment>
    );
}