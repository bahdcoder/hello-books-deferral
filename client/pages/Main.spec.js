import React from 'react';
import { shallow } from 'enzyme';
import Main from './Main';
describe('mount main', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Main debug />);

        expect(component).toMatchSnapshot();
    });
});
