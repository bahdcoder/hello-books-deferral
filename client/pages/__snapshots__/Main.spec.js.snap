// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount main should render correctly in "debug" mode 1`] = `
<Fragment>
  <div
    className="relative"
  >
    <BrowserRouter>
      <withRouter(App) />
    </BrowserRouter>
  </div>
</Fragment>
`;
